packages/components/psammead-image/src/index.amp.jsx

```diff
@@ -10,18 +10,20 @@ const AmpImg = props => {
 AmpImg.propTypes = {
   alt: string.isRequired,
   attribution: string,
-  height: number.isRequired,
+  height: number,
```

Comment:
Im afraid these need to be required as theyre a requirement set by google for all AMP images :(

Reply:
Ok

Reply:
Though I saw this - https://amp.dev/documentation/examples/style-layout/how_to_support_images_with_unknown_dimensions/ on supporting images with no dimensions. I'm thinking that having <AmpImg /> within a container that provides constraints fixes that. Like here - https://github.com/bbc/psammead/blob/9e7fab53ac4fe07fdcba7d9afd2aa6865fd6a993/packages/components/psammead-media-player/src/index.jsx#L69